lib/Wikia/src/Service/User/Auth/AuthService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Wikia\Service\User\Auth;
+
+interface AuthService {
```

Comment:
`Auth` was chosen so we can safely do both authentication and authorization without offending the name police.

I'm ok either splitting or combining. Since there is only one method should we wait to split until we add more functionality?